packages/editor/src/dataviews/actions/export-pattern.tsx

```diff
@@ -37,7 +37,7 @@ const exportPattern: Action< Pattern > = {
 	id: 'export-pattern',
 	label: __( 'Export as JSON' ),
 	supportsBulk: true,
-	isEligible: ( item ) => item.type === 'wp_block',
+	isEligible: ( item ) => item.type !== 'wp_template_part',
```

Comment:
This is to relax the check and allow non user patterns to be exported right? I tested a bit and didn't see any issues with this. I'm wondering though why we had that restriction before..

Reply:
I think it was probably a mistake that I did before when I ported this action. I don't see why we shouldn't be exporting "theme patterns". 

That said, ideally, we should have the same "type" for both theme and user patterns but that's a bigger/separate project.

Reply:
Actually there is an issue if you try to import such a pattern afterwards. I remember seeing some logic about `__file:pattern`..

Reply:
Interesting, ok, I'll remove that change then.